games/t.yaml

```diff
@@ -146,25 +146,33 @@
   updated: 2015-06-18
 
 - name: TES3MP
-  images:
-  - https://i2.wp.com/openmw.org/wp-content/uploads/2017/07/hA3oZhX.png
-  lang:
-  - C++
-  license:
-  - GPL3
-  development: halted
   originals:
   - 'The Elder Scrolls III: Morrowind'
-  repo: https://github.com/TES3MP/openmw-tes3mp
-  status: playable
   type: remake
+  repo: 'https://github.com/TES3MP/openmw-tes3mp'
+  url: 'http://tes3mp.com/'
+  development: active
+  status: playable
   multiplayer:
   - Online
-  url: http://tes3mp.com/
+  - LAN
+  content: swappable
+  lang:
+  - C++
+  license:
+  - GPL3
+  info: TES3MP is a project adding multiplayer functionality to OpenMW
+  updated: '2022-02-07'
+  images:
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/170828b.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-2983-11-27-07h06m58s509.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-4274-01-22-03h12m11s414.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-2432-09-08-08h21m43s889.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-7947-01-28-20h26m56s612.png
+  video:
+    moddb: 1907883
```

Comment:
there should only be one video tag, with both youtube and moddb

Reply:
Done in [d97152b](https://github.com/opengaming/osgameclones/pull/1623/commits/d97152b75462157f6261f5135d0fbfb38dd1b5c3).